### PR TITLE
Get and play bilibili video parts as songs in album

### PR DESCRIPTION
**Pull request by Wei-bin-Wu**
The album list of bilibili site was not working, because there is not album on bilibili, but I saw that there is the same search results list as results in single part, so I make those items to getting their video parts as an album, and then, making it playable.
 
Maybe this not be a final solution but it works fine for me now.

### CODE REVIEW

**Comment by mikelxk** [comment]
Thanks for your commit,
This commit enables to search audios in list of videos. Which is very cool.
The still so style error of the play button because most video cover is not a square.
image
I will take a lot at that as well

**Comment by Wei-bin-Wu** [comment]
> Thanks for your commit,
> This commit enables to search audios in list of videos. Which is very cool.
> The still so style error of the play button because most video cover is not a square.
> image
> I will take a lot at that as well

Thanks your reply, and I think that my work finishes there at the moment, it is just I said, works fine for me, because I use the extension on PC web and play songs on background, so I don’t care much for this approach which doesn’t affect the usability for me. 
The other reason could be that I’m not really good for front part, so it will take me more time. 
If you think these commits are useful, could you merge this PR? Then you could make your changes and improve the front part after merging. Or close it if your are just  developing something similar or planning to use this album part for other use case.
I will create new branch and PR whether I have new idea or changes in the future, thanks your so much.

**Review by mikelxk — APPROVED** [review]

**Comment by mikelxk** [comment]
I've submitted a pr to that just now. #976 
I've approved it and waiting for approval from @listen1

**Pull request merged**
